Makefile

```diff
@@ -117,24 +118,42 @@ ifneq ($(filter ppc64%,$(UNAME_M)),)
 	endif
 endif
 ifndef LLAMA_NO_ACCELERATE
-	# Mac M1 - include Accelerate framework.
-	# `-framework Accelerate` works on Mac Intel as well, with negliable performance boost (as of the predict time).
+	# Mac Intel & M1 - include Accelerate framework.
 	ifeq ($(UNAME_S),Darwin)
 		CFLAGS  += -DGGML_USE_ACCELERATE
 		LDFLAGS += -framework Accelerate
 	endif
 endif
 ifdef LLAMA_OPENBLAS
-	CFLAGS  += -DGGML_USE_OPENBLAS -I/usr/local/include/openblas -I/usr/include/openblas
-	ifneq ($(shell grep -e "Arch Linux" -e "ID_LIKE=arch" /etc/os-release 2>/dev/null),)
-		LDFLAGS += -lopenblas -lcblas
+	CFLAGS  += -DGGML_USE_OPENBLAS
+	ifeq ($(UNAME_S),Darwin)
+		# openblas installed with Homebew on macOS.
+		CFLAGS  += -I/usr/local/opt/openblas/include
+		LDFLAGS += -lopenblas -L/usr/local/opt/openblas/lib
 	else
-		LDFLAGS += -lopenblas
+		ifneq ($(shell grep -e "Ubuntu" -e "ID_LIKE=debian" /etc/os-release 2>/dev/null),)
```

Comment:
Let's try something else:
```bash
CFLAGS += $(pkg-config openblas --cflags)
LDFLAGS += $(pkg-config openblas --libs)
```

Reply:
It adds a dependency on `pkg-config` though. Linux etc. have it. w64devkit [has it](https://nullprogram.com/blog/2023/01/18/), what about macOS?

Another option would be to set all flags with `+=` which means the user can add their own on the make command line.

Reply:
I think the part that I mentioned is in non-Darwin branch.